src/erlfmt_cli.erl

```diff
@@ -305,8 +321,10 @@ resolve_parsed(PreferParsed, DefaultParsed) ->
             version;
         {_, version} ->
             version;
-        {{format, PreferFiles, PreferConfig}, {format, DefaultFiles, DefaultConfig}} ->
+        {{format, PreferFiles, PreferExclude, PreferConfig},
+            {format, DefaultFiles, DefaultExclude, DefaultConfig}} ->
             {format, resolve_files(PreferFiles, DefaultFiles),
+                resolve_files(PreferExclude, DefaultExclude),
                 resolve_config(PreferConfig, DefaultConfig)}
```

Comment:
Is this correct? Should one excludes apply to the file list from the other source?

Reply:
Yes I think this is correct, but maybe you have an example we haven't thought of